src/components/Onboarding/Screens/GardenTypeSelector.js

```diff
@@ -1,20 +1,144 @@
-import React from 'react'
+import React, { useCallback, useState } from 'react'
+import { GU, textStyle, useTheme } from '@1hive/1hive-ui'
 import { useOnboardingState } from '@providers/Onboarding'
 import Navigation from '../Navigation'
 
+import { BYOT_TYPE, NATIVE_TYPE } from '../constants'
+import defaultGardenLogo from '@assets/defaultGardenLogo.png'
+
 function GardenTypeSelector() {
-  const { onBack, onNext } = useOnboardingState()
+  const theme = useTheme()
+  const { config, onBack, onConfigChange, onNext } = useOnboardingState()
+  const [selectedType, setSelectedType] = useState(config.garden.type)
+
+  const handleSelectNative = useCallback(() => {
+    setSelectedType(NATIVE_TYPE)
+  }, [])
+
+  const handleSelectBYOT = useCallback(() => {
+    setSelectedType(BYOT_TYPE)
+  }, [])
+
+  const handleNext = useCallback(() => {
+    onConfigChange('garden', { type: selectedType })
+    onNext()
+  }, [onConfigChange, onNext, selectedType])
 
   return (
     <div>
-      Garden type
+      <div
+        css={`
+          margin-bottom: ${6 * GU}px;
+          text-align: center;
+        `}
+      >
+        <h1
+          css={`
+            ${textStyle('title1')};
+            margin-bottom: ${3 * GU}px;
+          `}
+        >
+          Garden type
+        </h1>
+        <div>
+          <p
+            css={`
+              ${textStyle('body1')};
+              color: ${theme.contentSecondary};
+            `}
+          >
+            Choose which type of garden you'd like to launch.
+          </p>
+        </div>
+      </div>
+      <div
+        css={`
+          display: flex;
+          align-items: center;
+          justify-content: space-around;
+          margin-bottom: ${6 * GU}px;
+        `}
+      >
+        <Card
```

Comment:
Wonder what you think of also add the `handleNext` handler to the `onSelect` prop of each `Card`?

Reply:
Ah yes i thought about it but wasn't sure, let's do it 👍